dependencies.yaml

```diff
@@ -503,3 +503,4 @@ dependencies:
           - pandas
           - *scikit_learn
           - seaborn
+          - xgboost
```

Comment:
Should probably be `py-xgboost`

Reply:
xgboost should be more than enough since it installs all what's needed afaict